dbms/src/Interpreters/sortBlock.cpp

```diff
@@ -14,15 +15,21 @@ namespace ErrorCodes
 }
 
 
-static inline bool needCollation(const IColumn * column, const SortColumnDescription & description)
+static inline const IColumn * needCollation(const IColumn * column, const SortColumnDescription & description)
```

Comment:
Now function's semantic has changes, but function name looks like it is still bool.